main.py

```diff
@@ -52,8 +53,22 @@ def on_focus_out2(event):
    tax_rate1.bind("<FocusIn>", on_entry_click2)
    tax_rate1.bind("<FocusOut>", on_focus_out2)
    tax_rate1.pack(pady=[0,10])
-
-   calc_start = Button(frame2, text='Take me to the Calculator',  command=result_window,)
+   
+   def checkval() -> None:
```

Comment:
I would add some validation that instantly informs the user about an invalid character and which textbox it is refering to